### PR TITLE
Clarify instructions

### LINKED ISSUE #36270: it says open .vscode/launch.json - but gives no clue as to where the file is and how to do it!
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 40d7fd04-e358-9a83-aef2-bfcba1169699
* Version Independent ID: f5df932b-b092-2832-9c9a-61a9c43dcdb3
* Content: [Debug a .NET console application using Visual Studio Code - .NET](https://learn.microsoft.com/en-au/dotnet/core/tutorials/debugging-with-visual-studio-code)
* Content Source: [docs/core/tutorials/debugging-with-visual-studio-code.md](https://github.com/dotnet/docs/blob/main/docs/core/tutorials/debugging-with-visual-studio-code.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #36270

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/tutorials/debugging-with-visual-studio-code.md](https://github.com/dotnet/docs/blob/e081ebb745a29b8680abd6b9f7e195d98f13b242/docs/core/tutorials/debugging-with-visual-studio-code.md) | [Tutorial: Debug a .NET console application using Visual Studio Code](https://review.learn.microsoft.com/en-us/dotnet/core/tutorials/debugging-with-visual-studio-code?branch=pr-en-us-36281) |

### CODE REVIEW

File: docs/core/tutorials/debugging-with-visual-studio-code.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ A *breakpoint* temporarily interrupts the execution of the application before th
 
 The breakpoint is located after a `Console.ReadLine` method call. The **Debug Console** doesn't accept terminal input for a running program. To handle terminal input while debugging, you can use the integrated terminal (one of the Visual Studio Code windows) or an external terminal. For this tutorial, you use the integrated terminal.
 
-1. Open *.vscode/launch.json*.
+1. The project folder contains a .vscode folder. Open the *launch.json* file that is located in the *.vscode* folder.
```
Comment (tdykstra) [inline docs/core/tutorials/debugging-with-visual-studio-code.md:46]:
```suggestion
1. The project folder contains a *.vscode* folder. Open the *launch.json* file that is in the *.vscode* folder.
```

File: docs/core/tutorials/debugging-with-visual-studio-code.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ A *breakpoint* temporarily interrupts the execution of the application before th
 
 The breakpoint is located after a `Console.ReadLine` method call. The **Debug Console** doesn't accept terminal input for a running program. To handle terminal input while debugging, you can use the integrated terminal (one of the Visual Studio Code windows) or an external terminal. For this tutorial, you use the integrated terminal.
 
-1. Open *.vscode/launch.json*.
+1. The project folder contains a *.vscode* folder. Open the *launch.json* file that is in the *.vscode* folder.
 
 1. Change the `console` setting from `internalConsole` to `integratedTerminal`:
```
Comment (tdykstra) [inline docs/core/tutorials/debugging-with-visual-studio-code.md:48]:
```suggestion
1. In *launch.settings*, change the `console` setting from `internalConsole` to `integratedTerminal`:
```

File: docs/core/tutorials/debugging-with-visual-studio-code.md [hunk 3]
```diff
@@ -43,9 +43,9 @@ A *breakpoint* temporarily interrupts the execution of the application before th
 
 The breakpoint is located after a `Console.ReadLine` method call. The **Debug Console** doesn't accept terminal input for a running program. To handle terminal input while debugging, you can use the integrated terminal (one of the Visual Studio Code windows) or an external terminal. For this tutorial, you use the integrated terminal.
 
-1. Open *.vscode/launch.json*.
+1. The project folder contains a *.vscode* folder. Open the *launch.json* file that is in the *.vscode* folder.
```
Comment (gewarren) [inline docs/core/tutorials/debugging-with-visual-studio-code.md:46]:
```suggestion
1. The project folder contains a *.vscode* folder. Open the *launch.json* file that's in the *.vscode* folder.
```

File: docs/core/tutorials/debugging-with-visual-studio-code.md [hunk 4]
```diff
@@ -43,9 +43,9 @@ A *breakpoint* temporarily interrupts the execution of the application before th
...
 
-1. Change the `console` setting from `internalConsole` to `integratedTerminal`:
+1. In *launch.settings*, change the `console` setting from `internalConsole` to `integratedTerminal`:
```
Comment (gewarren) [inline docs/core/tutorials/debugging-with-visual-studio-code.md:48]:
```suggestion
1. In *launch.json*, change the `console` setting from `internalConsole` to `integratedTerminal`:
```

**Review by gewarren — APPROVED** [review]
Was a bit confused but not sure if it actually should be launch.settings.

**Comment by tdykstra** [comment]
@gewarren Thanks for reviewing and catching that mistake!

**Pull request merged**